components/automate-cli/cmd/chef-automate/ca.go

```diff
@@ -126,3 +190,131 @@ func runRegenRootCmd(*cobra.Command, []string) error {
 	writer.Title("Certificate authority regenerated.")
 	return nil
 }
+
+func runInternalCaHA(infra *AutomateHAInfraDetails, scriptCommands string) error {
+	sshConfig := getSshDetails(infra)
+	if caCmdFlags.automate && caCmdFlags.chef_server && caCmdFlags.node != "" {
```

Comment:
Can we put this check in isFEFlagEnabled Function?

Reply:
done